doc/rst/technotes/gpu.rst

```diff
@@ -202,20 +223,21 @@ Chapel’s build system will automatically try and deduce what type of GPU you
 have and where your installation of relevant runtime (e.g. CUDA or ROCm) are.
 If the type of GPU is not detected you may set the ``CHPL_GPU`` environment
 variable manually to either ``nvidia`` or ``amd``.  ``CHPL_GPU`` may also
-manually be set to ``cpu`` to use `CPU as Device mode`_.
+manually be set to ``cpu`` to use `CPU-as-Device mode`_.
 
 Based on the value of ``CHPL_GPU``, Chapel's build system will also attempt to
 automatically detect the path to the relevant runtime. If it is not
 automatically detected (or you would like to use a different installation) you
 may set ``CHPL_CUDA_PATH`` and/or ``CHPL_ROCM_PATH`` explicitly.
 
-The ``CHPL_GPU_ARCH`` environment variable can be set to control the desired
-GPU architecture to compile for. The default value is ``sm_60`` for
+The ``CHPL_GPU_ARCH`` environment variable can be set to control the desired GPU
+architecture to compile for. The default value is ``sm_60`` for
 ``CHPL_GPU_CODEGEN=cuda``. You may also use the ``--gpu-arch`` compiler flag to
-set GPU architecture.  If using AMD, `this table in the LLVM documentation
-<https://llvm.org/docs/AMDGPUUsage.html#processors>`_ has possible architecture
-values (see the "processor" column). For NVIDIA, see the `CUDA Programming
-Guide
+set GPU architecture.  If using AMD, this variable must be set. `This table in
+the ROCm documentation
+<https://rocm.docs.amd.com/en/latest/release/gpu_os_support.html#linux-supported-gpus>`_
+has possible architecture values (see the "LLVM Target" column). For NVIDIA, see
```

Comment:
> This table in
the ROCm documentation
<https://rocm.docs.amd.com/en/latest/release/gpu_os_support.html#linux-supported-gpus>`_
has possible architecture values (see the "LLVM Target" column).

Side note (not anything we can/should do about it in this PR): 

but it'd probably be worth thinking/researching about if we could either automate this or if there's something better we could tell users to do. On our latest machine, this table didn't help me and I more-or-less ended up having to figure out the GPU arch by trial-and-error.

Reply:
Spoke too soon (should have actually checked your link), didn't notice that you moved it from the LLVM instructions (until I went back and read your OP).

Ok, this new URL is great.